### ISSUE TITLE
AP scores are zero on all classes on custom dataset

### ISSUE THREAD

**Issue by curiousgradient** [xuannianz/keras-CenterNet#37]
I am getting the AP scores as 0 on all classes on the validation set of my custom dataset (using --compute-val-loss). I have tried reducing the initial learning rate but it still remains zero. Any thoughts on why this is happening ?
